### DISCUSSION TITLE
Integrating Apache Server with Pixi Environment (Using WSGI?)

### DISCUSSION THREAD

**Discussion by gepz** [prefix-dev/pixi#1432] [Q&A]
Hi all,
I'm using pixi for managing projects and want to integrate it with an Apache server (by using WSGI?). Has anyone done this successfully?
Or is it currently best to pass requests between Python servers and Apache servers, or should I start the server within a pixi shell specific to one project?

**Comment by wolfv** [comment]
Hey @gepz that sounds interesting!

I haven't done this. But I would be interested in helping. Do you already have an idea on how to get started on this? There is also an `nginx` package on conda-forge, I believe.

**Reply by gepz** [reply]
Thanks for the offer! After some research, I found a way to integrate Pixi with Apache server using mod_proxy_uwsgi, as uWSGI supports configurations to run different scripts before starting a WSGI app instance for each app in different Pixi projects, so one could probably run a Pixi activation script.
